src/Propel/Runtime/ActiveQuery/ModelCriteria.php

```diff
@@ -440,29 +434,50 @@ public function select($columnArray)
         if (empty($columnArray)) {
             throw new PropelException('You must ask for at least one column');
         }
+        $this->isSelfSelected = true;
+        if ($this->formatter === null) {
+            $this->setFormatter('\Propel\Runtime\Formatter\SimpleArrayFormatter');
```

Comment:
minor, but we I guess we can use `SimpleArrayFormatter::class` here

Reply:
That's code I just moved around, but you are right, I should have replaced it (and now I have)